### PR TITLE
Add storybook-launcher extension

### PR DESCRIPTION
**Pull request by winkerVSbecks**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/storybook-launcher/src/components/DeleteStorybookAction.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Action, Icon, confirmAlert } from "@raycast/api";
+
+export default function DeleteStorybookAction(props: { id: string; onDelete: (id: string) => void }) {
+  return (
+    <Action
+      icon={Icon.Trash}
+      title="Delete Storybook"
+      shortcut={{ modifiers: ["ctrl"], key: "x" }}
```
Comment (pernielsentikaer) [inline extensions/storybook-launcher/src/components/DeleteStorybookAction.tsx:8]:
```suggestion
      shortcut={{ modifiers: ["ctrl"], key: "x" }}
      style={Action.Style.Destructive}
```

File: extensions/storybook-launcher/src/components/DeleteStorybookAction.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      onAction={async () => {
+        if (await confirmAlert({ title: "Are you sure?" })) {
```
Comment (pernielsentikaer) [inline extensions/storybook-launcher/src/components/DeleteStorybookAction.tsx:10]: You can add [style and icon](https://developers.raycast.com/api-reference/feedback/alert#properties) here

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by winkerVSbecks** [comment]
@pernielsentikaer thanks for the feedback. Updated.

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**